h2o-core/src/main/java/water/Scope.java

```diff
@@ -92,4 +89,7 @@ static public void untrack(Iterable<Key<Vec>> keys) {
     HashSet<Key> xkeys = scope._keys.peek();
     for (Key<Vec> key : keys) xkeys.remove(key); // Untrack key
   }
+
+  @Override
```

Comment:
is this used anywhere in the code of the core or algos? I don't think we should add features that are not used or are used just in tests